### DISCUSSION TITLE
Permission Based Expiration Times

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#230] [Ideas]
I'd like to be able to set different expiration timers for different ranks on my server, that way new players will have their claims deleted sooner (to help with hundreds of new players a day creating claims and never returning).

I think a permission based system would be best, like: **griefprevention.expiration.90** where the # is the number of days to expiration.

**Comment by RoboMWM** [comment]
To anyone who wants to implement this, note that permissions only apply to players who are online. May have to use vault or related for offline permissions (if that's even supported)?
